swagger.yaml

```diff
@@ -6419,6 +6419,11 @@ definitions:
         x-nullable: false
         default: 'x86_64'
         description: The CPU architecture of the image (x86_64/arm64/etc).
+      discovery_kernel_parameters:
+        type: string
+        x-nullable: true
+        x-go-custom-tag: gorm:"type:text"
+        description: Json formatted string array representing the discovery image kernel parameters.
```

Comment:
nit: `JSON` and not `Json` because we already use the former around our API description, so we should not introduce a new way